website/content/docs/platform/k8s/vso/sources/vault/client-cache.mdx

```diff
@@ -0,0 +1,266 @@
+---
+layout: docs
+page_title: Enable the encrypted Vault client cache
+description: >-
+  Learn how to enable the operator's encrypted Vault Client cache for dynamic secrets
+---
+
+# Enable the encrypted Vault client cache
+
+The Vault Secrets Operator supports storing and encrypting its Vault client cache using Vault Server's [transit secrets engine](/vault/docs/secrets/transit). By default client cache storage is not enabled, since it requires extra Vault Server setup, but this guide will walk you through the steps to enable and configure it.
+
+<Highlight>
+
+Enabling encrypted client cache storage is recommended especially when using [VaultDynamicSecrets](/vault/docs/platform/k8s/vso/api-reference#vaultdynamicsecret).
+
+</Highlight>
+
+## Step 1: Configure transit
+
+Use the following commands/terraform to enable the transit secret engine, a key for the client cache, and a policy for the operator to use the key.
```

Comment:
nit: Here and elsewhere, should we capitalise Operator to help make clear it's the named product, and not the generic term for a human user? Or alternatively should we establish the VSO initialism at the top and use that everywhere?

```suggestion
Use the following commands/terraform to enable the transit secret engine, a key for the client cache, and a policy for the Operator to use the key.
```

Reply:
Yeah I added a "(VSO)" near the top, and Sarah's recommendations removed most of the other "operator" references.